Resources/Prototypes/Recipes/Crafting/Graphs/bots/mulebot.yml

```diff
@@ -0,0 +1,29 @@
+- type: constructionGraph
+  id: MuleBot
+  start: start
+  graph:
+  - node: start
+    edges:
+    - to: bot
+      steps:
+      - tag: ProximitySensor
+        icon:
+          sprite: Objects/Misc/proximity_sensor.rsi
+          state: icon
+        name: proximity sensor
+      - tag: BorgHead
+        icon:
+          sprite: Objects/Specific/Robotics/cyborg_parts.rsi
+          state: borg_head
+        name: borg head
+        doAfter: 1
+      - tag: BorgArm
```

Comment:
for what? if it has no arms eventually

Reply:
Youre right, it was again, carried over from all other lesser borgs (cleanbot etc) which all require at a minimum, one borg arm. I have removed it.